packages/manager/modules/manager-pci-common/src/components/tiles-input-choice/TileChoice.component.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { PropsWithChildren } from 'react';
+import clsx from 'clsx';
+import {
+  selectableTileClass,
+  selectedTileClass,
+  tileBorderClass,
+  tileDisabledClass,
+} from '../../constants/style';
+
+export interface TileChoiceProps {
+  isSelected?: boolean;
+  isDisabled?: boolean;
+}
+export function TileChoice({
```

Comment:
I don't understand the purpose of this, could you rename it ?

Reply:
<img width="872" alt="Capture d’écran 2025-02-10 à 13 21 01" src="https://github.com/user-attachments/assets/bd37be50-4787-4912-a4a3-fec11999673c" />

the purpose is a tile selector like the one in the screenshot (with generic tile content), which better name would you suggest ?

Reply:
it contains all the specific behavior of pci tile selection such as a specific single tile display once the choice is submitted, it also handle the disability of tiles (both of thoses are currently not handled in mrc)

i think it will be nice to not have to style a mrc component each time you want to display a tile choice in pci 

this component is currently used in object storage uapp so removing it from this PR will also imply to having to refactor this uapp